### ISSUE TITLE
can't kiss my npcs anymore </3

### ISSUE THREAD

**Issue by leximundo** [Norbyte/bg3se#136]
since patch 4 whenever i smooch my in game boyfriend it skips right over the kiss and exits me out of the dialog.

**Comment by Fancy-Toast** [comment]
Same issue, not sure if it's a bug with the update or a bug with the mods.  I will say I've uninstalled all my mods and it still isn't working so waiting to see if Larian Studios posts an update about this.  Seems like a lot of people are having this issue actually.

**Comment by Fancy-Toast** [comment]
UPDATE If you use BG Mod Manager for your mods, launch the game through there and the kiss animations work.  If you're using Vortex, not sure if that works.  TBH though I've found BG Mod Manager works better than Vortex.
![tempsnip](https://github.com/Norbyte/bg3se/assets/149851983/70ab4432-afc3-4f3d-8ddf-056bc580ad9d)

**Comment by Kali492** [comment]
Launching the game via BG Mod Manager didn't solve this for me.

There may be other mods interfering with this though - there's an ongoing discussion on the Nexus page of Mod Fixer if it might be responsible for the broken kiss animations. In my case the scenes are skipped even with all mods deleted, so might as well be a vanilla bug introduced with the patch.

**Comment by sleepines** [comment]
i can't get the kiss scenes either and i'm not using mod fixer..

**Comment by sleepines** [comment]
ok i opened the game thru mod manager and i finally see them, but the game seemed really glitchy and something this way O.o

**Comment by Kali492** [comment]
After trying for the last two days I got it to work again, somehow. Tried a lot, so not exactly sure what did the trick. So, some ideas for everyone else out there trying to troubleshoot this for themselves right now:

- purge all mods
- delete (or rename) the BG3 folder in your appdata->local (_this purges the savegame_; so definitely either backup manually or let steam backup via cloud; it does however force steam to reload all the configuration files)
- start the game again (to recreate the configuration files)
- if not already installed: get BG3 Mod Manager, reinstall mods over there
- update the script extender over the MG3 Mod Manager toolbox
- start the game - but not over steam; either via mod manager or just use the exe file in your game folder
- if that doesn't work: use the dx11 exe
- **load an earlier save**

Using a few or one of those steps at a time didn't work unfortunately, neither starting it via BG3MM, nor loading an earlier save, not even purging all mods and clean installing them, but after all those steps combined something seems to have worked and as long as I don't use the lari launcher but literally any other means to start the game it loads fine and the kiss animations work again.

There seem to be some minor bugs with voice lines cutting our too early, but since this is a known vanilla bug I don't think any mod behavior would change it.

**Comment by average-noodles** [comment]
Going with Kali492's suggestion I found a potential fix.

The issue seems to be stemming from having an old version of the script extender. However, updating the script extender does not fix the issue in saves after the first time you kissed your chosen companion. Meaning, you'll have to reload a save from before then.

Currently, if I went back, I would lose close to 9 hours. Hoping a future update from either Norbyte or Larian can fix this.
